Source/JavaScriptCore/runtime/ISO8601.cpp

```diff
@@ -1748,7 +1748,7 @@ static Int128 roundTimeDurationToIncrement(JSGlobalObject* globalObject, Int128
 }
 
 // https://tc39.es/proposal-temporal/#sec-temporal-roundtimeduration
-static Int128 roundTimeDuration(JSGlobalObject* globalObject, Int128 timeDuration, unsigned increment,
+Int128 roundTimeDuration(JSGlobalObject* globalObject, Int128 timeDuration, unsigned increment,
     TemporalUnit unit, RoundingMode roundingMode)
```

Comment:
Ditto.

Reply:
Fixed in [66fe9a7](https://github.com/WebKit/WebKit/pull/50388/commits/66fe9a740f5b5c2e6e103fb24c5924d3fa2b8735)